lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -586,12 +586,12 @@ def upload_grades_csv(_xmodule_instance_args, _entry_id, course_id, _task_input,
             cohorts_group_name = []
             if course.is_cohorted:
                 group = get_cohort(student, course_id, assign=False)
-                cohorts_group_name.append(group.name if group else '')
+                cohorts_group_name.append(unicode(group.name) if group else u'')
```

Comment:
I figured it could be either - though currently, we set cohort names in `cohort_handler`, which looks at `request.json.get('name')`, which should always return a unicode object. Does that make it safe to assume it'll always be unicode?